tools/run_tests/xds_k8s_test_driver/tests/outlier_detection_test.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2022 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import logging
+from typing import List
+
+from absl import flags
+from absl.testing import absltest
+
+from framework import xds_k8s_testcase
+from framework import xds_url_map_testcase
+
+logger = logging.getLogger(__name__)
+flags.adopt_module_key_flags(xds_k8s_testcase)
+
+# Type aliases
+RpcTypeUnaryCall = xds_url_map_testcase.RpcTypeUnaryCall
+RpcTypeEmptyCall = xds_url_map_testcase.RpcTypeEmptyCall
+_XdsTestServer = xds_k8s_testcase.XdsTestServer
+_XdsTestClient = xds_k8s_testcase.XdsTestClient
+
+# Testing consts
+_QPS = 100
+_REPLICA_COUNT = 5
+
+
+class OutlierDetectionTest(xds_k8s_testcase.RegularXdsKubernetesTestCase):
```

Comment:
Would be amazing to add a brief comment here and link https://github.com/grpc/grpc/blob/master/doc/xds-test-descriptions.md#outlier_detection.

I know we don't do this in other places, but it's a good time to start being more descriptive in our tests.

The body of the comment could be the same as in the test spec:

> This test verifies that the client applies the outlier detection configuration and temporarily drops traffic to a server that fails requests.

Reply:
Added